### ISSUE TITLE
It always escape the whole line

### ISSUE THREAD

**Issue by ruisilva450** [milovidov983/EscapeQuotes#2]
Steps to reproduce:
1. Have a json file. Eg:
```
{
  "questionText": { "en-US": "Marketed Brands?" }
}
```
2. Select ```{ "en-US": "Marketed Brands?" }```
3. Right click and select **Escape quotes**

The json comes as:
```
{
  \"questionText\": { \"en-US\": \"Marketed Brands?\" }
}
```

The expected result:
```
{
  "questionText": { \"en-US\": \"Marketed Brands?\" }
}
```
I know that this is not valid json but this is for me to later add the additional quotes wrapping the escaped bit.

**Comment by milovidov983** [comment]
I saw the problem, as soon as I have time, I will solve it.

**Comment by milovidov983** [comment]
The problem is fixed in version 1.3.3
